src/util.js

```diff
@@ -58,6 +60,14 @@ export function to(promise) {
     .catch(err => [err]);
 }
 
+export function getPatronBlocks(manualBlocks, automatedBlocks) {
+  let manualPatronBlocks = manualBlocks.filter(p => p.borrowing === true) || [];
```

Comment:
Could you please rename p to matuaBlock and automatedBlock?

Reply:
Disagree. This is local usage of variable, short name can be used. Thanks for review.